tests/sample_schema.pyi

```diff
@@ -9,8 +9,16 @@ import typing_extensions as typ
 # Definitions
 @typ.final
 class DictDefinition(typ.TypedDict):
+    """
+    Title
+    
+    A description of the schema.
+    
+    Examples: [{'nested_ref': 's010_cave', 'number': 0}, {'nested_ref': 's020_magma', 'number': 1}]
```

Comment:
Wouls rather have each example in it's own line, without being wrapped in a list